### PR TITLE
Update the kube-vip to v0.4.4

### LINKED ISSUE #628: Adopt latest kube-vip for powervs template
/kind feature
/area provider/ibmcloud

**Describe the solution you'd like**
Currently [v0.3.7](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/32400fcf34bc7869b492d96df28b408b90474121/templates/cluster-template-powervs.yaml#L130) is used and there are many releases post that, lets use the latest available version

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

### PR DESCRIPTION
**Pull request by mkumatag**
**What this PR does / why we need it**:

Used the doc here - https://kube-vip.chipzoller.dev/docs/installation/static/#generating-a-manifest to generate the manifest and updated it accordingly.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #628 

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: templates/cluster-template-powervs.yaml [hunk 1]
```diff
@@ -103,10 +103,10 @@ spec:
               env:
               - name: vip_arp
                 value: "true"
-              - name: vip_interface
-                value: "env2"
               - name: port
                 value: "6443"
+              - name: vip_interface
+                value: env2
```
Comment (Prajyot-Parab) [inline templates/cluster-template-powervs.yaml:109]: env2 -> "env2" just a observation, if both works fine then we are good.

Comment (mkumatag) [reply templates/cluster-template-powervs.yaml:109]: yes, its not really required.

**Pull request merged**